test/Microsoft.TestPlatform.Common.UnitTests/ExtensionFramework/Utilities/RunSettingsProviderExtensionsTests.cs

```diff
@@ -230,6 +236,17 @@ public void QueryRunSettingsNodeShouldReturnCorrectValue()
             Assert.AreEqual("x86", this.runSettingsProvider.QueryRunSettingsNode("RunConfiguration.TargetPlatform"));
         }
 
+        private void CheckRunSettingsAreUpdated(string parameterName, string parameterValue)
+        {
+            var match = this.runSettingsProvider.GetTestRunParameterNodeMatch($"TestRunParameters.Parameter(name=\"{parameterName}\",value=\"{parameterValue}\")");
+            var runSettingsWithTestRunParameters = $"<?xml version=\"1.0\" encoding=\"utf-16\"?>\r\n<RunSettings>\r\n  <TestRunParameters>\r\n    <Parameter name=\"{parameterName}\" value=\"{parameterValue}\" />\r\n  </TestRunParameters>\r\n</RunSettings>";
```

Comment:
Could you split it to lines and then string.join with environment.newline? That would make it a bit more readable.

Reply:
Done.